microsoft/PowerToys
# [FancyZones] Open window on active monitor

## Summary of the Pull Request
By default most newly created windows are opened on primary desktop (certain applications introduce their own custom positioning). If we have zone history for some application on secondary monitor, and that monitor is active, application window will be opened there, otherwise we fallback to default windows behavior and that window is opened on primary monitor. This PR introduces new functionality that will open window on secondary monitor (if secondary monitor is active) even if there is no application zone history.


## PR Checklist
* [x] Applies to #4357


## Validation Steps Performed
Use multi monitor configuration, and preferably different scaling on different monitors. Try opening same application on different monitors. Application should not be resized in any way unless window can't fit the screen. Also, try with portrait / landscape positioned monitors.
